.flake8

```diff
@@ -0,0 +1,5 @@
+[flake8]
+# Recommend matching the black line length (default 88)
+max-line-length = 88
+exclude=.venv,.git
+ignore = E203, E501, W503
```

Comment:
Flake8 is unfortunately the only tool I use that does not currently support `pyproject.toml`-based configuration. That said, my preference would be to put this in `setup.cfg`, just to cut down on file clutter.

Also, are you aware of specific reasons we would need the `exclude` directive? I would think Flake8 is smart enough not to process the `.git` directory, and the `.venv` exclude would be better placed in a global configuration file on developers' individual workstations, since not everyone puts virtual environments inside project directories. (All of mine are in: `~/virtualenvs`)

Finally, I don't think we need the comment here. It should be clear from our other configuration what the line-length convention is.

Reply:
If we include in pre-commit think we can remove it maybe?